Combobox.js

```diff
@@ -575,22 +571,50 @@ define([
 			return new ComboPopup({combobox: this});
 		},
 
+		evaluateFilteringConditions: function (inputElement) {
+			var mobile = this._useCenteredDropDown();
+			if (this.minFilterChars !== inputElement.value.length !== 0 &&
```

Comment:
That's exactly what I meant when I was saying to be careful during the review. In fact I'm not fully convinced neither and I think the function could be rewritten in a better/elegant way.